### ISSUE TITLE
[Bug] W1008 rule cannot be configured

### ISSUE THREAD

**Issue by rikerfi** [MarketSquare/robotframework-robocop#793]
Rule W1008 cannot be configured to the Error nor Info severity. Both command line and argument file configurations are failing.

Robocop version: 2.8.1

**Comment by mnojek** [comment]
Hey @rikerfi 
Can you please write an example of the command for running robocop that fails?
Can you also include what kind of error do you get?

**Comment by rikerfi** [comment]
```python
# *** Settings ***
Documentation     Example


*** Test Cases ***
A Log Test
  Log  Hello
    Log  Hi
```
Commands:
`robocop koe.robot --configure 1008:severity:I`
output
`[W] 1008 Line is over-indented (bad-indent)`
expected
`[I] 1008 Line is over-indented (bad-indent)`

`robocop koe.robot --configure 1008:severity:E`
output
`[W] 1008 Line is over-indented (bad-indent)`
expected
`[E] 1008 Line is over-indented (bad-indent)`

However, it seems that run is severity aware for the rule when configure severity to Info and using threshold Warning:
`robocop koe.robot --configure 1008:severity:I -t W`

Above command removes rule 1008 from output as it should be.

**Comment by mnojek** [comment]
Thanks for more info, @rikerfi!
I found the issue. I will fix it and release in the next Robocop 2.9.0.
